### PR TITLE
Multilingual support, cite encycl. & dict, URL ...

### PR DESCRIPTION
**Pull request by bellerophons-pegasus**
Extensive changes in CSL to allow for use in multiple languages (esp. EN, DE, IT, FR and ES) with some custom locale. Added citation of Encyclopedia Article and Dictionary Entry. Added display of URLs and DOIs (line break with display block is not working properly). Added event and event date for conference papers. Added translators. Added review. Enforce use of Journal abbreviations if present. Added Presentation. Enhanced citation of Thesis. Ancient Authors: using short title as a work around (until a custom item type will be introduced in Zotero)

### CODE REVIEW

File: deutsches-archaologisches-institut.csl [hunk 1]
```diff
@@ -274,42 +298,118 @@
...
-  <macro name="series">
+  <macro name="doi">
     <choose>
-      <if match="any" variable="collection-title">
-        <text variable="collection-title" strip-periods="true" prefix=", "/>
-        <text variable="collection-number" prefix=" "/>
+      <if match="any" variable="DOI">
+        <text value=", "/>
+        <group display="block">
+          <text value="doi: " font-variant="small-caps"/>
```
[the remark this answers is not in the document]
Comment (bellerophons-pegasus) [reply deutsches-archaologisches-institut.csl]: The official documentation on the DAI homepage does not state anything about how to use DOIs. But in the bibtex style (https://github.com/LukasCBossert/biblatex-archaeologie) that is how they are dealing with it.

File: deutsches-archaologisches-institut.csl [hunk 2]
```diff
@@ -237,26 +245,42 @@
...
+                  <text variable="original-publisher-place"/>
+                  <date date-parts="year" form="text" variable="original-date" prefix=" "/>
+                  <text value="; "/>
+                </if>
+              </choose>
+              <text variable="publisher-place"/>
+            </else>
+          </choose>
+          <choose>
+            <if match="none" is-uncertain-date="issued">
```
[the remark this answers is not in the document]
Comment (bellerophons-pegasus) [reply deutsches-archaologisches-institut.csl]: I reworked the style using the documentation of an extensive bibtex style (https://github.com/LukasCBossert/biblatex-archaeologie) officially supported by the DAI and tried to get all given examples in the right format. 
What I am testing here is if the date is not uncertain (i.e. the issued date is set), to be able to render the parentheses correctly. But maybe I just got on the wrong track and this is not necessary. I'll check that.

File: deutsches-archaologisches-institut.csl [hunk 3]
```diff
@@ -274,42 +298,118 @@
...
+    <choose>
+      <if type="paper-conference" match="any">
+        <text variable="volume" prefix=" "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="url">
+    <choose>
+      <if match="any" variable="URL">
+        <text value=", "/>
```
[the remark this answers is not in the document]
Comment (bellerophons-pegasus) [reply deutsches-archaologisches-institut.csl]: Solved

File: deutsches-archaologisches-institut.csl (hunk 2 above)
[the remark this answers is not in the document]
Comment (bellerophons-pegasus) [reply deutsches-archaologisches-institut.csl]: Yeah, was too deep into that. Removed the condition.

[the remark this answers is not in the document]
Comment (adam3smith) [reply deutsches-archaologisches-institut.csl]: Thanks -- just to clarify: `is-uncertain` doesn't refer to a date not being set, but refers to a date being imprecise (such as `c.1063`). To just test for the presence of the date `<if variable="issued"/>` is all you need, but note that CSL will simply not render variables that don't exist, so I don't think you even need that here.

File: deutsches-archaologisches-institut.csl [hunk 4]
```diff
@@ -339,18 +438,35 @@
...
+            <text macro="secondary-contributors"/>
+            <text macro="review"/>
+            <text macro="short-container-contributors"/>
+            <text macro="locators"/>
+            <text macro="issue"/>
+            <text macro="series"/>
+            <choose>
+              <if type="article article-journal article-magazine article-newspaper" match="none">
+                <text macro="translator"/>
+                <text value=" "/>
```
[the remark this answers is not in the document]
Comment (bellerophons-pegasus) [reply deutsches-archaologisches-institut.csl]: Solved

File: deutsches-archaologisches-institut.csl (hunk 2 above)
[the remark this answers is not in the document]
Comment (bellerophons-pegasus) [reply deutsches-archaologisches-institut.csl]: Is there anything I can do to signal github and you, that I made the changes requested?

[the remark this answers is not in the document]
Comment (adam3smith) [reply deutsches-archaologisches-institut.csl]: I get auto-updated on your changes, so a clear commit messages (such as the one you used) is all you need, thanks; I just didn't have time to final review & merge last night.

**Comment by adam3smith** [comment]
Thanks!

**Comment by bellerophons-pegasus** [comment]
Thank you too!

**Comment by adam3smith** [comment]
@bellerophons-pegasus there is a recent report of locators not appearing in citations for the DAI style. I haven't tested this, but would you mind taking a look? https://forums.zotero.org/discussion/71355/seitenzahlen-werden-nach-aktualisierung-nicht-mehr-angezeigt#latest

**Comment by bellerophons-pegasus** [comment]
Yes, I'll have a look!

**Comment by bellerophons-pegasus** [comment]
I accidentally removed the locator part in the inline citations. Added it again in a new pull request.

**Pull request merged**